charts/platform/Chart.yaml

```
@@ -0,0 +1,36 @@
+apiVersion: v2
+name: platform
+description: Helm chart for installing the CloudQuery self-hosted platform
+home: www.cloudquery.io
+icon: https://cloudquery.io/assets/img/logo.png
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+maintainers:
+  - name: mnorbury
+    email: martin@cloudquery.io
```

```suggestion
    email: info@cloudquery.io
```

If this email works or any kind of generic email I think (e.g. support@cloudquery.io) assuming this will be used to contact us?

---

The email change works, but the `name` field needs to be a real github user name. We have `cq-bot`, or maybe create a `cq-info` github account.

---

Well https://github.com/cloudquery/helm-charts/blob/e74cc1bf504977dc8ad4fab3ca5679c24369724b/charts/cloudquery/Chart.yaml#L13